addons/spreadsheet/static/tests/widgets/spreadsheet_binary_field.test.js

```diff
@@ -0,0 +1,32 @@
+import { expect, test } from "@odoo/hoot";
+import { defineModels, fields, models, mountView, onRpc } from "@web/../tests/web_test_helpers";
+import { defineMailModels } from "@mail/../tests/mail_test_helpers";
+import { click } from "@odoo/hoot-dom";
+import { animationFrame } from "@odoo/hoot-mock";
+
+const BINARY_FILE = "R0lGODlhDAMAKIFAF5LAP/zxANyuAP/gaP///wACH5BAEAUALAw";
+
+class SpreadsheetDashboard extends models.Model {
```

Comment:
Already defined in `odoo\addons\spreadsheet_dashboard\static\tests\helpers\data.js`.

Reply:
Any reason you create a new binary field `document` instead of testing on the existing `spreadsheet_binary_data` ?

Reply:
Sorry I did not ignore your comments here, but when I saw the first comment I just went straight to talk with Francois 😉 

I was not aware of the existence of this model, I will use it then thank 🙁